migrations/0003-databaseuser.sql

```diff
@@ -0,0 +1,9 @@
+-- -----------------------------------------------------
+-- Table `animalkeeping`.`DatabaseUserType`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `animal_keeping`.`DatabaseUserType` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(200) NOT NULL,
+  `privileges` VARCHAR(500) NOT NULL,
+  PRIMARY KEY (`id`)) 
+ENGINE = InnoDB;
```

Comment:
another thing,  the user is not linked to the person. I suggest to add a "login" or "user" field to the person. Otherwise we can not link any changes from the user to the person.